design/jira-traceability.md

```diff
@@ -120,3 +120,4 @@ integrations:
     - production     
     environmentType: '{"stage-US":"staging"; "East-US": "production";}' 
 ```
+ 
```

Comment:
Few comments on stages and environmentype, we should not keep the environmentype with the integration, we should keep it with individual job, what ever the name they want. Main problem with this approach is, if someone change stage name and they do not change in the integration, then we have to cross verify etc etc problem.

Also, what is the security modal, if I can make changes to YAML file that is good enough or should I have permission on the jiraconnection end point?

Other confusion in the example YAML is:
stages: 
    - build
    - production     
    environmentType: '{"stage-US":"staging"; "East-US": "production";}' 

we are saying stages as build/production and in environment type we have use ‘east-us’ etc names, how they are co-related I am not sure, full yaml be helpful.